oonipipeline/src/oonipipeline/transforms/measurement_transformer.py

```diff
@@ -359,6 +338,44 @@ def find_tls_handshake_network_events(
     return None
 
 
+def find_tls_handshake_network_events_with_transaction_id(
+    tls_handshake: TLSHandshake,
+    network_events: Optional[List[NetworkEvent]],
+) -> Optional[List[NetworkEvent]]:
+    transaction_map = {}
+    all_event_windows_dict = {}
+
+    for ne in network_events:
+        if ne.transaction_id is None:
+            log.warning("detected network event without transaction_id")
```

Comment:
Same here, the more info we can extract to debug and troubleshoot the bug, I think the better.